src/targets/gpu/problem_cache.cpp

```diff
@@ -24,11 +24,34 @@
  */
 #include <migraphx/gpu/problem_cache.hpp>
 #include <migraphx/ranges.hpp>
+#include <migraphx/json.hpp>
+#include <migraphx/env.hpp>
+#include <migraphx/serialize.hpp>
+#include <migraphx/file_buffer.hpp>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace gpu {
 
+MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_PROBLEM_CACHE)
+
+void problem_cache::load()
+{
+    auto pc_path = string_value_of(MIGRAPHX_PROBLEM_CACHE{});
+    if(pc_path.empty())
+        return;
+    if(not fs::exists(pc_path))
+        return;
```

Comment:
It would be nice to print out warning/log message saying path is non-existent.

Reply:
added.